internal/bundle/bundle_ext.go

```diff
@@ -166,3 +167,26 @@ func (x *RiskMagnitude) UnmarshalYAML(node *yaml.Node) error {
 	}
 	return nil
 }
+
+func (x *HumanTime) UnmarshalYAML(node *yaml.Node) error {
```

Comment:
I think we also need to implement MarshalYAML here as we've removed this earlier